components/service/glean/metrics.yaml

```diff
@@ -209,6 +209,8 @@ glean.internal.metrics:
 glean.error:
   invalid_value:
     type: labeled_counter
+    send_in_pings:
```

Comment:
This will need a version bump [here](https://github.com/mozilla-mobile/android-components/blob/6eb9b2c4b0854d62e6ae1e3255b5ed5e97060ccb/components/service/glean/scripts/sdk_generator.gradle#L24)